environment_mac.yml

```diff
@@ -122,7 +122,7 @@ dependencies:
   - openjpeg=2.3.1=h254dc36_3
   - openpyxl=3.0.3=py_0
   - openssl=1.1.1g=h0b31af3_0
-  - osmnx=0.10=py_1
+  - osmnx=0.11.4=py_1
```

Comment:
Are you on a mac? We should probably update this for all operating systems, although I'm fine leaving that for later. I do want to get the mac tests fixed though before we merge this because at one point updating the osmnx version was separately causing tests to fail and I want to make sure that's not still the case.

That's on me though since it's a preexisting issue; I'll try to get that done this week.

Reply:
Oh, whoops, I didn't see that it's only this out of date on mac, so that's not relevant. Still want to get the tests working first though.